core/comment_events.js

```diff
@@ -96,6 +106,23 @@ Blockly.Events.CommentBase.prototype.toJson = function() {
 Blockly.Events.CommentBase.prototype.fromJson = function(json) {
   this.commentId = json['commentId'];
   this.group = json['group'];
+  this.blockId = json['blockId'];
+};
+
+/**
+ * Helper function for finding the comment this event pertains to.
+ * @return {Blockly.WorkspaceComment | Blockly.ScratchBlockComment}
+ *     The comment this event pertains to.
+ * @private
+ */
+Blockly.Events.CommentBase.prototype.getComment_ = function() {
+  var workspace = this.getEventWorkspace_();
+  if (this.blockId) {
+    var block = workspace.getBlockById(this.blockId);
+    if (block) return block.comment;
```

Comment:
Nit: return on a new line, and braces, even for simple stuff like this.  Here and throughout the PR.